pages/CONTRIBUTING-docs.md

```diff
@@ -0,0 +1,72 @@
+# JSON Schema documentation contributing guide 
+
+This is guide describes the workflow to contribute to the JSON Schema technical documentation. If you would like to contribute to other aspects of the JSON Schema website, like writing blog articles or publishing a case study, please read the website's [Contributing Guide](/website/CONTRIBUTING.md) for more details. 
+
+## Welcome
+Thank you for helping us improve our documentation.
+
+This guide is intended for use by documentation contributors and will help you get started with the basics of our working processes.
+
+## Before you start
+To contribute to the JSON Schema documentation you will need to complete the following steps:
+
+1. [Create a GitHub account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github), if you don't have one already.
+2. Read our [Code of Conduct](https://json-schema.org/overview/code-of-conduct).
+3. [Install the JSON Schema website](https://github.com/json-schema-org/website/blob/main/INSTALLATION.md) on your computer.
+4. Read our style guide. 
+
+## Community engagement
+To stay updated with news about the JSON Schema project and its documentation, engage with our team, ask questions, and discuss ideas, please join our [Slack workspace](https://json-schema.org/slack). We particularly recommend the following channels:
+
+- #documentation: Discussions related to the JSON Schema documentation.
+
+- #website: Discussions related to our website.
+
+- #introductions: Self-introductions and connecting with other members of the community.
+
+- #community-announcements: Events and updates related to the JSON Schema project.
+
+- #anouncements: Important announcement from the JSON Schema organization.
+
+- #stack-overflow: Stack overflow questions tagged with JSON Schema.
+  
+## Best practices
+
+This is a list of best practices the JSON Schema project strives and resources we recommend for additional information:
+
+1. File and branching naming conventions: We use dashes instead of spaces, and lowercase letters to name .md files and branches. For example: _implementers-guide.md_, _web-feat-diataxis_.
+2. Commit messages: To learn how to write good commit messages, see [Step-by-step guide on how to write good commit messages on Git](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/).
+
+## Contribution workflow 
+We value documentation and appreciate efforts to help us improve our documents. There are two ways you can contribute to improve our documentation:
```

Comment:
We value documentation and appreciate **your** efforts to help us improve our documents